### ISSUE TITLE
Remove ads in code

### ISSUE THREAD

**Issue by cryptoprof** [vuepress-reco/vuepress-theme-reco#304]
#### What problem does this feature solve?
I have started use this theme, but in some cases adblock blocks ads from this theme
http://r.analytic.press/?orig=
and some others

#### How should this be implemented in your opinion?
Have an option to disable ads?

**Comment by recoluan** [comment]
Hello, please describe your problem in detail. First of all, I can't open your link and cannot reproduce your problem, and there is no advertisement in the theme. You can attach a screenshot to which piece of content you are referring to.

**Comment by cryptoprof** [comment]
Hello and thank you for this great theme. I have created my blog recently.
Url: [http://cryptoprof.ru](http://cryptoprof.ru)
my package.json is next:
```
{
  "name": "Cryptoprof",
  "version": "1.0.0",
  "author": "reco_luan",
  "scripts": {
    "dev": "vuepress dev . --open --host \"localhost\"",
    "build": "vuepress build ."
  },
  "devDependencies": {
    "vuepress": "1.7.1",
    "vuepress-theme-reco": "1.6.1"
  },
  "description": "Мой новый блог"
}
```
I added some pages and do 
`yarn run build`
After that i deploy files to my github pages.
Hm...
I found the reason. It is in my provider. I'm sorry for wasting your time. 
I'm from Russia and reason of my ads is injection of the ads content from provider side.
Article from habr: https://habr.com/ru/company/roskomsvoboda/blog/485902/
If i use https://cryptoprof.ru all works awesome. Thank you again for awesome theme!
